# Set this to 'true' to pause new measurements immediately after reset.
# Note: This is a set behaviour and not a toggle behaviour, contrary to Pause.srv service.
bool pause_new_measurements false
---
# Result code defintions
uint8 RESULT_SUCCESS=0

uint8 result
